# Sense service: Data driven guided

string logFileName    # Name of the logFile before sensing the state of the hand (relative to logFolder)
string singulationModelName      # Name of the model to compare against.
string poseEstimationModelName   # Name of the model to compare against.
---
bool singulated       # 1-yes 0-no.
int64 nMarkers        # number of markers.
float64 confidence    # confidence of the output.
float64 x             # x coordinate of the center of the marker (in the tool reference frame).
float64 y	      # y coordinate of the center of the marker (in the tool reference frame).
float64 theta         # orientation of the marker in the image
float64 alpha         # angle of line perpendicular to marker axis
float64 distance      # distance from the axis of the marker to the center of the palm (origin of tool reference frame).
string logFileName    # name of the logFile after appending the output of the sense service (relative to logFolder).
int64 ret
string msg
